### ISSUE TITLE
No tests.

### ISSUE THREAD

**Issue by ndobromirov** [codemix/yii2-streamlog#6]
The tool is starting to collect functionality, but there are no automated tests for it.

**Comment by mikehaertl** [comment]
@ndobromirov That's right. Feel free to add something. Only guidelines: I prefer simple solutions. If I had time I'd probaly use our [dockerized yii2](https://github.com/codemix/yii2-dockerized) app. Just not sure how to best integrate this with tests in travis or how to check for `stdout` / `stderr`. Before you spend too much time, maybe we can first discuss the approach here.
